### PR TITLE
Add section on how to add the initial user in Quarkus distribution.

### LINKED ISSUE #283: Migration Guide for Quarkus doesn't contain information on initial user setup
### Describe the bug

The migration guide doesn't note that there is no longer a shell script to add the initial user.

### Version

17.0.0

### Expected behavior

The migration guide should mention this.

### Actual behavior

https://www.keycloak.org/migration/migrating-to-quarkus

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ahus1**
Also fixing a link and formatting.

Closes #283

### CODE REVIEW

File: guides/migration/migrating-to-quarkus.adoc [hunk 1]
```diff
@@ -40,16 +40,25 @@ We recommend that you do this build step as a part of installing Keycloak, throu
 
 However, there is also an auto build mode that makes Keycloak behave more or less the same as the WildFly distribution in this regard. This comes with a startup time penalty, but is still able to optimise the runtime better than the WildFly distribution could.
 
-## Default context path changed
+== Setup of initial users
 
-By default the new Quarkus distribution removes `/auth` from the context-path. To re-introduce the `/auth` use the `http-relative-path`[https://www.keycloak.org/server/all-config?q=http-relative-path&f=build] build option. For example:
+The Keycloak Wildfly distribution contained scripts like `add-user-keycloak.sh` to add initial users to keycloak.
```
Comment (jonkoops) [inline guides/migration/migrating-to-quarkus.adoc:45]:
```suggestion
The Keycloak Wildfly distribution contained scripts named `add-user-keycloak.sh` to add initial users to Keycloak.
```

File: guides/migration/migrating-to-quarkus.adoc [hunk 2]
```diff
@@ -40,16 +40,25 @@ We recommend that you do this build step as a part of installing Keycloak, throu
...
+These are no longer included in the Quarkus distribution.
+
+To add the initial admin user, set the environment variables `KEYCLOAK_ADMIN` and `KEYCLOAK_ADMIN_PASSWORD` for username and password of the user.
```
Comment (jonkoops) [inline guides/migration/migrating-to-quarkus.adoc:48]:
```suggestion
To add the initial admin user, set the environment variables `KEYCLOAK_ADMIN` and `KEYCLOAK_ADMIN_PASSWORD` for the username and password of the user.
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by ahus1** [comment]
@jonkoops - thanks, accepted your changed and squashed them. Please re-review to approve.

**Review by jonkoops — APPROVED** [review]
LGTM

**Review by stianst — APPROVED** [review]

**Pull request merged**